README.md

```diff
@@ -1,2 +1,136 @@
-# consume-more-stuff
-A custom CMS for personal product sale
+# Consume More Stuff
+> A custom CMS for personal product sale
+
+## Introduction
+Build a Content Management System using:
+
+- __Backend__
+    - **Express** as the application server
+    - **Postgres** as the database server
+    - **Sequelize.js** as the ORM
+    - **passport.js** for Authentication and Authorization
+
+- __Frontend__
+	- **React.js** or **AngularJS** as FE framework
+	- **Sass** for styling
+	- **Webpack** for bundling	
```

Comment:
Small question, but, if they are using webpack with AngularJS, should they be using ES6? If so, this might involve a different, class based approach to developing their front end...which would be awesome.

https://github.com/yannickglt/angular-webpack-boilerplate
https://github.com/kitconcept/webpack-starter-angular

Reply:
We use webpack AngularJS generator in class. We would just have them use that.

Reply:
Ah cool 👍